diffblue.yml

```diff
@@ -1,4 +1,2 @@
-branch: master
-language: java
-buildCmd: "mvn build"
-testCmd: "mvn cobertura:cobertura"
+buildCmd: "mvn test"
```

Comment:
If it's complaining about a missing dependency, `mvn test` must only be compiling a subset of the code, which means we will only generate tests for that subset, which, ironically, will be the part of the code that already has tests.

We need to fix the dependency issue.